lib/rails/generators/mongoid/config/templates/mongoid.yml

```diff
@@ -118,17 +118,32 @@ development:
         # The file containing concatenated certificate authority certificates
         # used to validate certs passed from the other end of the connection.
         # ssl_ca_cert: /path/to/ca.cert
-        
+
         # Whether to truncate long log lines. (default: true)
         # truncate_logs: true
 
-  # Configure Mongoid specific options. (optional)
+  # Configure Mongoid-specific options. (optional)
   options:
 <%- Mongoid::Config::Introspection.options.each do |opt| -%>
     <%= opt.indented_comment(indent: 4) %>
     # <%= opt.name %>: <%= opt.default %>
 
 <%- end -%>
+  # Configure Driver-specific options. (optional)
+  driver_options:
+    # When this flag is off, an aggregation done on a view will be executed over
+    # the documents included in that view, instead of all documents in the
+    # collection. When this flag is on, the view fiter is ignored.
+    # broken_view_aggregate: true
+
+    # When this flag is set to false, the view options will be correctly
+    # propagated to readable methods.
+    # broken_view_options: true
+
+    # When this flag is set to true, the update and replace methods will
+    # validate the paramters and raise an error if they are invalid.
+    # validate_update_replace: false
+
```

Comment:
Would make this a single whitespace line rather than double

Reply:
It probably comes down to personal preference at this point, here. I tend to prefer a stronger delineation between environment sections in these config files, especially if one of the sections is large (like the `development` section in the template). A wider gap between sections makes it easier to see at a glance where one section ends and the other begins.